extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1136,6 +1141,12 @@ export class AmpA4A extends AMP.BaseElement {
             'src': srcPath,
             'name': nameData,
           }, SHARED_IFRAME_PROPERTIES));
+      if (method == XORIGIN_MODE.NAMEFRAME){
+        const sentinel = generateSentinel(window);
+        const metadata = getContextMetadata(window, iframe, sentinel);
+        let name = JSON.stringify({creative, _context: metadata._context});
```

Comment:
`const`, since it doesn't change. Also, I'd prefer for these string keys to be quoted; even if the actual Closure rules (which I don't fully understand) don't allow renaming of fields under these circumstances, it's better for no one reading the code to have to wonder about that.